scripts/test

```diff
@@ -5,22 +5,12 @@ if [ -d 'venv' ] ; then
     export PREFIX="venv/bin/"
 fi
 
+set -ex
+
 if [ -z $GITHUB_ACTIONS ]; then
-    set +e
     scripts/check
-    while [ $? -ne 0 ]; do
-        read -p "Running 'scripts/check' failed. Do you want to run 'scripts/lint' now? [y/N]  " yn
```

Comment:
Removed this just like we did with `httpx` https://github.com/encode/httpx/pull/1107